src/lib/containers/Login.tsx

```diff
@@ -17,8 +18,6 @@ type State = {
 }
 
 type Props = {
-  theme: string
-  icon: boolean
```

Comment:
Note: this is a breaking change.
* `theme` only seemed to change the color of the button. I imagine if we are going to try to create a dark theme (e.g. [SWC-5241](https://sagebionetworks.jira.com/browse/SWC-5241)), then we would probably implement it slightly differently.
* `icon` didn't do anything except manipulate padding inside a <ButtonContent>

Reply:
double check that styleguidist isn't using these props